content/applications/sales/sales/send_quotations/quote_template.rst

```diff
@@ -2,41 +2,43 @@
 Quotation templates
 ===================
 
-In Odoo *Sales*, salespeople have the ability to create reusable quotation templates for common
-products or services that the business offers.
+Reusable quotation templates can be made in Odoo's **Sales** app for common products or services.
 
-By using these templates, quotations can be tailored and sent to customers at a much faster pace,
+By using these templates, quotations can be tailored and sent to customers at a quicker pace,
 without having to create new quotations from scratch every time a sales negotiation occurs.
 
 .. _sales/send_quotations/templates:
 
 Configuration
 =============
 
-Begin by activating the setting in :menuselection:`Sales app --> Configuration --> Settings`, and
-scroll to the :guilabel:`Quotations \& Orders` heading.
+To use quotation templates, begin by activating the setting in :menuselection:`Sales app -->
+Configuration --> Settings`, and scroll to the :guilabel:`Quotations \& Orders` heading.
 
-In that section, check the box beside the :guilabel:`Quotation Templates` option. Doing so reveals a
-new :guilabel:`Default Template` field, in which a default quotation template can be chosen from a
+Under the heading, tick the :guilabel:`Quotation Templates` checkbox. Doing so reveals a new
+:guilabel:`Default Template` field, in which a default quotation template can be chosen from the
 drop-down menu.
 
 .. image:: quote_template/quotations-templates-setting.png
    :align: center
    :alt: How to enable quotation templates on Odoo Sales.
 
-Also, upon activating the :guilabel:`Quotation Template` feature, an internal :guilabel:`➡️
-Quotation Templates` link appears beneath the :guilabel:`Default Template` field.
+Upon activating the :guilabel:`Quotation Template` feature, an internal :icon:`fa-arrow-right`
+:guilabel:`(Quotation Templates)` link appears beneath the :guilabel:`Default Template` field.
```

Comment:
No need to include the parenthesis when a UI element has text and an icon (like smartbuttons)
```suggestion
:guilabel:`Quotation Templates` link appears beneath the :guilabel:`Default Template` field.
```